### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by vivianjeng**
## Pull request checklist

Please check if your PR fulfills the following requirements:

- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [x] Build (`yarn build`) was run locally and any changes were pushed
- [x] Lint (`yarn lint --check`) has passed locally and any fixes were made for failures

## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?

- Update function descriptions
- Update `README.md`
- Update `package.json` descriptions

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by iamonuwa — APPROVED** [review]
LGTM!

**Review by AnthonyMadia — COMMENTED** [review]
Looks good

**Review by AnthonyMadia — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,30 +1,70 @@
...
+        <img alt="Github license" src="https://img.shields.io/github/license/unirep/unirep.svg?style=flat-square">
+    </a>
+    <a href="https://eslint.org/">
+        <img alt="Linter eslint" src="https://img.shields.io/badge/linter-eslint-8080f2?style=flat-square&logo=eslint">
+    </a>
+    <a href="https://prettier.io/">
+        <img alt="Code style prettier" src="https://img.shields.io/badge/code%20style-prettier-f8bc45?style=flat-square&logo=prettier">
+    </a>
+    <a href="https://dl.circleci.com/status-badge/redirect/gh/Unirep/Unirep/tree/develop">
+        <img alt="Circle CI" src="https://dl.circleci.com/status-badge/img/gh/Unirep/Unirep/tree/develop.svg?style=svg">
```
Comment (vimwitch) [inline README.md:19]: Build badge needs to be updated. Consider using the flat one from shields to match the others:

CircleCI

**Review by vimwitch — COMMENTED** [review]
I'm not a huge fan of inline documentation comments. They're only useful if you run them through a processor to generate a static site. But even then, the static site usually isn't much help.

If we want good docs (and we do if we want people integrating) then we need a custom site that details the exports of each package and each function/object. A good example documentation site is [ethers](https://docs.ethers.io/v5/api/). The [cloudflare workers](https://developers.cloudflare.com/workers/) docs are another good example.

**Comment by vivianjeng** [comment]
> I'm not a huge fan of inline documentation comments. They're only useful if you run them through a processor to generate a static site. But even then, the static site usually isn't much help.

I think it is helpful for development. Yeah but if it should be synchronized with documentation site.

I build the [package group](https://unirep.gitbook.io/unirep/package/crypto) in the document.
we can specified function/object there.

**Pull request merged**